server/metrics/requests.go

```diff
@@ -36,6 +36,7 @@ func (g *RequestEndpointMetadata) GetPreInitializedTags() map[string]string {
 	return map[string]string{
 		"method":       g.methodInfo.Name,
 		"grpc_service": g.serviceName,
+		"namespace":    "default_namespace",
```

Comment:
nit: not for this diff, but probably we can define this constant somewhere(may be in types.go), currently it is defined in the metadata package.